lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
> OK I'll go ahead and depend on snusnu/cookie then. Currently, cookie doesn't support any cookie header validation, although I'd love to implement that at some point.

Can we make it so that you at least get `nil` returned when the cookie is formatted in an invalid way?